spec/models/manageiq/providers/redhat/infra_manager/vm_import_spec.rb

```diff
@@ -11,7 +11,7 @@
   let(:cluster_path_escaped)  { 'Folder1%2FFolder%20%40%23%24*2%2FCompute%203%2FFolder4%2FCluster%205' }
 
   let(:cluster_guid) { cluster.uid_ems }
-  let(:storage_guid) { storage.ems_ref_obj.split('/').last }
+  let(:storage_guid) { storage.ems_ref.split('/').last }
```

Comment:
@agrare LGTM, but storage_redhat factory is not defining ems_ref, so specs are failing.
I can see that there's an open PR in the core that removes ems_ref_obj from the factory but not replacing with ems_ref  https://github.com/ManageIQ/manageiq/pull/19430

Reply:
:+1: thanks @gekorob I'll add that to the factory in the core PR and add this repo to the cross-repo tests PR

Reply:
@agrare great. So can I merge it it is it ok?

Reply:
Since it depends on https://github.com/ManageIQ/manageiq/pull/19430 it'll have to be merged together

Reply:
Ok